ui/cocoa/view.m

```diff
@@ -180,6 +180,20 @@ static CGRect compute_cursor_clip_rect(int screen_height,
     return rect;
 }
 
+static CGEventRef handleTapEvent(CGEventTapProxy proxy, CGEventType type, CGEventRef cgEvent, void *userInfo)
+{
+
```

Comment:
Empty lines are not present at the beginning of other functions. Please remove it.

Reply:
Definitely a mistake, probably after removing a debug print statement. Will do.